sky/serve/service_spec.py

```diff
@@ -23,8 +23,12 @@ def __init__(
         max_replicas: Optional[int] = None,
         qps_upper_threshold: Optional[float] = None,
         qps_lower_threshold: Optional[float] = None,
+        target_qps_per_replica: Optional[float] = None,
         post_data: Optional[Dict[str, Any]] = None,
-        auto_restart: bool = True,
+        auto_restart: Optional[bool] = None,
+        autoscaling_decision_interval: int = 20,
+        upscale_delay_seconds: int = 300,
+        downscale_delay_seconds: int = 1200,
```

Comment:
nit: Use constants for these values; also use default value = None and change to default value in `__init__`